tests/conftest.py

```diff
@@ -4,14 +4,35 @@
 from app.db import get_session
 from fastapi.testclient import TestClient
 
+from app.auth.security import get_password_hash
+from app.models.users import Users
+
+SECRET_KEY = "TEST_KEY"
+
 
 @pytest.fixture(name="session")
 def session_fixture():
     engine = create_engine(
         "sqlite://", connect_args={"check_same_thread": False}, poolclass=StaticPool
     )
     SQLModel.metadata.create_all(engine)
+    users_to_add = [
+        {"username": "cla_admin", "password": "cla_admin", "disabled": False},
+        {"username": "jane_doe", "password": "password", "disabled": True},
+    ]
     with Session(engine) as session:
+        for user in users_to_add:
+            username = user.get("username")
+            password = user.get("password")
+            disabled = user.get("disabled")
+
+            password = get_password_hash(password)
+            new_user = Users(
+                username=username, hashed_password=password, disabled=disabled
```

Comment:
This can be much simpler by writing 
```python
password = user.get("password")
user["password"] = get_password_hash(password)

new_user = Users(**user)
```